administration/src/mui-modules/activation/components/ActivationPageContent.tsx

```diff
@@ -0,0 +1,37 @@
+import { Button } from '@mui/material'
+import { styled } from '@mui/system'
+import React, { ReactElement } from 'react'
+import { useLocation } from 'react-router-dom'
+
+import ProjectSwitcher from '../../../bp-modules/util/ProjectSwitcher'
+import InvalidActivationLink from '../../../errors/templates/InvalidActivationLink'
+import { ProjectConfig } from '../../../project-configs/getProjectConfig'
+
+const ActivationButton = styled(Button)`
+  margin-top: 20px;
+`
+
+const ActivationPageContent = ({ config }: { config: ProjectConfig }): ReactElement => {
+  const { hash } = useLocation()
+
+  if (!config.activation) {
+    return <ProjectSwitcher />
+  }
+
+  const { activationText, downloadLink } = config.activation
+
+  if (!hash) {
+    return <InvalidActivationLink />
+  }
+
+  return (
+    <>
+      {activationText(config.name, downloadLink)}
+      <ActivationButton variant='outlined' color='primary' href={window.location.href}>
+        Aktiviere Karte
+      </ActivationButton>
```

Comment:
Did you test whether this button actually works?
* Uninstall the app
* Load this activation page in the browser
* Install the app
* Go back to the browser
* Click on link

We should probably also test this in the three different browsers, i.e. Firefox, Chrome, Safari
I suspect that it might not work in every browser :/

Reply:
Alternatively, we can also just remove this button and tell the user to go back and click the link again.

Reply:
Gonna test this. Why I added this link was because when you click on the qr code of the pdf it does not open new tab/window and the user has to go back after app installation. I mean we need this feature for Koblenz so it would be really bad if the 3 big browsers don't support it. I mean this is the main use case for DeepLinking

Reply:
Ah okay now I got the issue. This is a good point Koblenz. Gonna test this

Reply:
ok i tested this. In chrome in works out of the box but in other browsers you have to "allow open links with external apps" f.e. firefox. So i think you solution to reopen the pdf is the better way